src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -36,7 +36,7 @@ public class AddCommand extends Command {
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_MEETING + "2024-02-12 "
+            + PREFIX_MEETING + "2024-07-12 12:00"
```

Comment:
good addition for this so that people know that the use case we need to have HH:MM